usecases/schema/manager.go

```diff
@@ -415,48 +413,3 @@ func (m *Manager) ResolveParentNodes(class, shardName string) (map[string]string
 	}
 	return name2Addr, nil
 }
-
-func (m *Manager) Nodes() []string {
```

Comment:
Both `Nodes()` and `nodeName()` functions appear to be unrelated to the schema. I would recommend extracting them into a new specialized class